src/Microsoft.OData.Core/ODataInnerError.cs

```diff
@@ -21,19 +20,20 @@ namespace Microsoft.OData
     [DebuggerDisplay("{Message}")]
     public sealed class ODataInnerError
     {
-        /// <summary>Initializes a new instance of the <see cref="Microsoft.OData.ODataInnerError" /> class with default values.</summary>
+        /// <summary>Initializes a new instance of the <see cref="ODataInnerError" /> class with default values.</summary>
         public ODataInnerError()
         {
-            Properties = new Dictionary<string, ODataValue>();
-
-            //// NOTE: we add empty elements if no information is provided for the message, error type and stack trace
-            ////       to stay compatible with Astoria.
-            Properties.Add(JsonConstants.ODataErrorInnerErrorMessageName, new ODataNullValue());
-            Properties.Add(JsonConstants.ODataErrorInnerErrorTypeNameName, new ODataNullValue());
-            Properties.Add(JsonConstants.ODataErrorInnerErrorStackTraceName, new ODataNullValue());
+            Properties = new Dictionary<string, ODataValue>
+            {
+                //// NOTE: we add empty elements if no information is provided for the message, error type and stack trace
+                ////       to stay compatible with Astoria.

```

Comment:
what's this Astoria. I thought it was some old thing that we moved away from?

Reply:
@ElizabethOkerio Astoria was codename for "WCF Data Services". There could still be services that depend on the behaviour associated with Astoria

Reply:
We are moving away from these WCF services as they are not supported in .net 8.0

Reply:
@ElizabethOkerio I have removed the logic to support Astoria compatibility. `message`, `type`, and `stacktrace` will no longer be added to the dictionary by default. I also dropped `Message`, `Type` and `StackTrace` properties since they were added specifically for Astoria compatibility. One can can add `message`, `type`, and `stacktrace` to the `Properties` dictionary is they want to preserve Astoria compatibility.